scripts/git/patches/uWebsockets/0001-fix-want-write-crash.patch

```diff
@@ -0,0 +1,57 @@
+From fe5f7b393233b95b2c5ba66f74de1d435124373e Mon Sep 17 00:00:00 2001
+From: "google.com" <google.com>
+Date: Fri, 2 Jul 2021 11:31:55 -0400
+Subject: [PATCH] fix want_write crash
+
+---
+ src/Socket.h | 30 ++++++++++++++++--------------
+ 1 file changed, 16 insertions(+), 14 deletions(-)
+
+diff --git a/src/Socket.h b/src/Socket.h
+index 2179ff8..a97360f 100644
+--- a/src/Socket.h
++++ b/src/Socket.h
+@@ -306,24 +306,26 @@ protected:
+         if (messageQueue.empty()) {
+ 
+             if (ssl) {
++              /* BEG Patched by firebase-cpp-sdk 00001-fix-want-write-crash.patch */
++              bool continue_loop = true;
++              do {
+                 sent = SSL_write(ssl, message->data, message->length);
+                 if (sent == (ssize_t) message->length) {
+-                    wasTransferred = false;
+-                    return true;
++                  wasTransferred = false;
++                  return true;
```

Comment:
Is it possible to use the original indentation to have a cleaner diff?

Reply:
Done.